Upstream mistagged 13.8 to the commit which opened development of the next
version (confusingly using 13.8 still, rather than bumping it).  Reported to
upstream here: https://github.com/wavefrontHQ/wavefront-proxy/issues/914

Author: Daniel Watkins <daniel.watkins@chainguard.dev>
--
diff --git a/proxy/pom.xml b/proxy/pom.xml
index d72a6c8..549e85e 100644
--- a/proxy/pom.xml
+++ b/proxy/pom.xml
@@ -4,7 +4,7 @@
 
   <groupId>com.wavefront</groupId>
   <artifactId>proxy</artifactId>
-  <version>13.8-RC1-SNAPSHOT</version>
+  <version>13.8</version>
 
   <name>Wavefront Proxy</name>
   <description>Service for batching and relaying metric traffic to Wavefront</description>
